src/js/lib/angularjs-autogrow.js

```diff
@@ -0,0 +1,98 @@
+(function(){
```

Comment:
I see that you checked-in the angularjs-autogrow library, presumably from https://github.com/evyros/angularjs-autogrow

This library is available as an npm library, but you've chosen to check it in.  This works because we (unfortunately) still have a way to include checked-in-dependencies outside of webpack.  Could you explain your rationale for choosing this method over the npm dependency, which would be preferred?

If we do end up checking it in, we should have a comment referencing where the code came from, as an internet citation or sorts.  e.g. https://github.com/evyros/angularjs-autogrow

Reply:
> I see that you checked-in the angularjs-autogrow library, presumably from https://github.com/evyros/angularjs-autogrow
> 
> This library is available as an npm library, but you've chosen to check it in. This works because we (unfortunately) still have a way to include checked-in-dependencies outside of webpack. Could you explain your rationale for choosing this method over the npm dependency, which would be preferred?
> 
> If we do end up checking it in, we should have a comment referencing where the code came from, as an internet citation or sorts. e.g. https://github.com/evyros/angularjs-autogrow

I made some changes to the library to allow passing in offset to the grow height.

Reply:
I added a comment in the forked library.